### DISCUSSION TITLE
performance about deit-tiny

### DISCUSSION THREAD

**Discussion by stlmx** [open-mmlab/mmclassification#1712] [Q&A]
Thanks for mmlab !
I notice that deit-tiny in mmpretrain is 74.5% while the performance in official deit is 72.2%.
I want to know the reason why the deit-tiny in mmpretrain performs better ?
Thanks a lot !

**Comment by Ezra-Yu** [comment]
Small models with a limited number of parameters generally do not require very strong data augmentation, and we have to replicate the data augmentation that officially attenuates training in comparison.
